examples/step-58/doc/intro.dox

```diff
@@ -0,0 +1,875 @@
+  \qquad\qquad\text{with initial condition}\;
+  \psi^{(2)}(t_n) &= \psi(t_n),
+\\
+  \frac{d\psi^{(3)}}{dt}
+  &=
+  -i\kappa |\psi^{(3)}|^2 \,\psi^{(3)},
+  &
+  \text{for}\; t \in (t_n,t_{n+1}),
+  \qquad\qquad\text{with initial condition}\;
+  \psi^{(3)}(t_n) &= \psi(t_n).
+@f}
+In other words, they are all trajectories $\psi^{(k)}$ that start at
+$\psi(t_n)$ and integrate up the effects of exactly one of the three
+terms. The increments resulting from each of these terms over our time
+interval are then $I^{(1)}=\psi^{(1)}(t_{n+1})-\psi(t_n)$,
+$I^{(2)}=\psi^{(2)}(t_{n+1})-\psi(t_n)$, and
+$I^{(3)}=\psi^{(3)}(t_{n+1})-\psi(t_n)$.
+
+It is now reasonable to assume (this is an approximation!) that the
+change due to all three of the effects in question is well approximated
+by the sum of the three separate increments:
+@f{align*}{
+ \psi(t_{n+1})-\psi(t_n)
+ \approx
+ I^{(1)} + I^{(2)} + I^{(3)}.
+@f}
+This intuition is indeed correct, though the approximation is not
+exact: the difference between the exact left hand side and the term
+$I^{(1)}+I^{(2)}+I^{(3)}$ (i.e., the difference between the <i>exact</i> increment
+for the exact solution $\psi(t)$ when moving from $t_n$ to $t_{n+1}$,
+and the increment composed of the three parts on the right hand side),
+is proportional to $\Delta t=t_{n+1}-t_{n}$. In other words, this
+approach introduces an error of size ${\cal O}(\Delta t)$. Nothing we
+have done so far has discretized anything in time or space, so the
+<i>overall</i> error is going to be ${\cal O}(\Delta t)$ plus whatever
+error we commit when approximating the integrals (the temporal
+discretization error) plus whatever error we commit when approximating
+the spatial dependencies of $\psi$ (the spatial error).
+
+Before we continue with discussions about operator splitting, let us
+talk about why one would even want to go this way? The answer is
+simple: For some of the separate equations for the $\psi^{(k)}$, we
+may have ways to solve them more efficiently than if we throw
+everything together and try to solve it at once. For example, and
+particularly pertinent in the current case: The equation for
+$\psi^{(3)}$, i.e.,
+@f{align*}{
+  \frac{d\psi^{(3)}}{dt}
+  &=
+  -i\kappa |\psi^{(3)}|^2 \,\psi^{(3)},
+  \qquad\qquad
+  &
+  \text{for}\; t \in (t_n,t_{n+1}),
+  \qquad\qquad\text{with initial condition}\;
+  \psi^{(3)}(t_n) &= \psi(t_n),
+@f}
+or equivalently,
+@f{align*}{
+  \psi^{(3)}(t_{n+1})
+  &=
+  \psi(t_n)
+  +
+  \int_{t_n}^{t_{n+1}}
+  \left(
+  -i\kappa |\psi^{(3)}(t)|^2 \,\psi^{(3)}(t)
+  \right)
+  \;
+  dt,
+@f}
+can be solved exactly: the equation is solved by
+@f{align*}{
+  \psi^{(3)}(t) = e^{-i\kappa|\psi(t_n)|^2 (t-t_{n})} \psi(t_n).
+@f}
+This is easy to see if (i) you plug this solution into the
+differential equation, and (ii) realize that the magnitude
+$|\psi^{(3)}|$ is constant, i.e., the term $|\psi(t_n)|^2$ in the
+exponent  is in fact equal to $|\psi^{(3)}(t)|^2$. In other words, the
```

Comment:
`exponent  is` -> `exponent is`

(double space, although I'm sure it'd render just fine)